File: src/main/kotlin/solutionsJava/MissingNumber3.java
import java.util.Scanner;

public class MissingNumber3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        String[] numbersLine = scanner.nextLine().split(" ");

        long sum = 0L;
        for (int i = 0; i < numbersLine.length; i++) {
            sum += Integer.parseInt(numbersLine[i]);
        }

        long actualSum = ((1L + n) * n) / 2;

        System.out.println(actualSum - sum);
    }
}

File: src/main/kotlin/solutions/MissingNumber3.kt
package uz.safix.solutions

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val n = scanner.nextInt(); scanner.nextLine()
    val numbersLine = scanner.nextLine().split(" ")

    var sum = 0L
    for (i in numbersLine.indices) { sum += numbersLine[i].toInt() }

    val actualSum = ((1L + n) * n) / 2

    println(actualSum - sum)
}